src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -41,12 +42,16 @@
 class AreaEncoder(json.JSONEncoder):
     """Convert the Area class hierarchy to json dict."""
     def default(self, o):
-        if isinstance(o, AreaBase):
+        # Leaf classes are Areas too, therefore the Area/AreaBase classes need to be handled
+        # separately.
+        if type(o) in [Area, AreaBase]:
```

Comment:
Just had a look into `isinstance` and I think you can also write:
```
isinstance(o, (AreaBase, Area)):
```
to achieve the same.

Reply:
It is unfortunately not the same, `isinstance(Leaf, AreaBase)` will return True which is not what we want here. Instead, we want the Leaf classes to be handled by a different method from the pure Area / AreaBase classes.

Reply:
Ahh. ok. Thanks!